code/src/index.css

```diff
@@ -1,13 +1,65 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  background: rgb(225, 222, 222);
+  font-family: "Trispace", sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.message-card,
+.input-card {
+  margin: 20px auto;
+  border: 2px solid black;
+  width: 500px;
+  box-shadow: 5px 5px;
+  padding: 20px;
+  background-color: #f2f0f0;
+}
+
+.heart-image {
+  height: 20px;
+  width: 20px;
+}
+
+.message-text {
+  word-wrap: break-word;
+}
+
+.message-heart {
+  display: flex;
+  flex-direction: column;
```

Comment:
I can't find this either. Is it used somewhere? Otherwise you should remove this